src/snyk/common/views/issueTreeProvider.ts

```diff
@@ -35,6 +38,14 @@ export abstract class ProductIssueTreeProvider<T> extends AnalysisTreeNodeProvid
     );
   }
 
+  getAIFix(issue: Issue<T>): boolean {
```

Comment:
do we want to keep this generic or enforce that it is only Code?

Does this make sense to be inside the abstract class? Why not keep this inside the Code implementation?